### ISSUE TITLE
Did you consider alternatives to Markdown?

### ISSUE THREAD

**Issue by davidrunger** [wahabs/Dulynote#6]
Maybe you intentionally want to go with Markdown, but this is a learning curve for the non-technically-savvy that would deter many potential users from your app. Have you considered something more along the lines of a WYSIWYG editor? Googling "text editor plugin jquery" brings up plenty of hits, for example. The first one that I looked at seems extremely simple to use, maybe moreso even than Markdown.

**Comment by wahabs** [comment]
I was actually hoping to do a kind of combination of the two (like the StackEdit editor) where you can manually put in Markdown but also click a button that will insert the formatting for you (e.g. clicking the Italics button will put the asterisks around the cursor). You might be right about accessibility though, I think I'll leave this open until I've done more research into text editor plugins.

**Comment by davidrunger** [comment]
Ahh, that would be cool. I clicked through on your link to StackEdit but I failed to notice those bold, italics, etc. buttons at the top of the page.

But yeah, like you say, usability/mass appeal is still a question. If you do decide to incorporate Markdown, then I'd both (1) make those style-related buttons more noticeable somehow than they are on StackEdit and (2) brand your site specifically as "Evernote for Markdown" or something so that employers don't think that the Markdown was just some weak solution what you resorted to in order to allow your users to create formatted text in an Evernote clone when you couldn't figure out a better way to do so.
